### ISSUE TITLE
Combined total for multiple Sliders

### ISSUE THREAD

**Issue by orion3dgames** [Venturocket/angular-slider#58]
I'm trying to implement a page where there are 4 sliders, and I want to make it so the combined total of all 4 sliders will never go over 100.

Is this possible?

**Comment by drgould** [comment]
One way you could do this currently is update the ceiling values for the other three sliders when one is moved. The downside to that approach though is the knobs of the other sliders will appear to move as the slider range is changed.

I'm currently in the process of completely rewriting the slider. I'll keep this in mind and see if I can figure out a better solution.

**Comment by orion3dgames** [comment]
Yes, that's the way I'm doing it at the moment, but I'm quite happy with the results. Feels a little cluncky thought.
Thanks for your answer and plugin :)
